crates/nu-command/src/filesystem/ls.rs

```diff
@@ -12,12 +12,27 @@ use nu_protocol::{
     PipelineMetadata, ShellError, Signature, Span, Spanned, SyntaxShape, Value,
 };
 use pathdiff::diff_paths;
+
+#[cfg(windows)]
+use std::mem::MaybeUninit;
 #[cfg(unix)]
 use std::os::unix::fs::PermissionsExt;
+#[cfg(windows)]
+use std::os::windows::prelude::OsStrExt;
 use std::path::PathBuf;
 use std::sync::Arc;
 use std::time::{SystemTime, UNIX_EPOCH};
 
+#[cfg(windows)]
+use windows::Win32::Foundation::FILETIME;
+#[cfg(windows)]
+use windows::Win32::Storage::FileSystem::{
+    FindFirstFileW, FILE_ATTRIBUTE_DIRECTORY, FILE_ATTRIBUTE_READONLY,
+    FILE_ATTRIBUTE_REPARSE_POINT, WIN32_FIND_DATAW,
+};
+#[cfg(windows)]
+use windows::Win32::System::SystemServices::{IO_REPARSE_TAG_MOUNT_POINT, IO_REPARSE_TAG_SYMLINK};
```

Comment:
What about using something like this to group up `#[cfg(windows)]`? 
```rust
#[cfg(windows)]
use {
    std::mem::MaybeUninit,
    std::os::windows::prelude::OsStrExt,
    windows::Win32::Foundation::FILETIME,
    windows::Win32::Storage::FileSystem::{
        FindFirstFileW, FILE_ATTRIBUTE_DIRECTORY, FILE_ATTRIBUTE_READONLY,
        FILE_ATTRIBUTE_REPARSE_POINT, WIN32_FIND_DATAW,
    },
    windows::Win32::System::SystemServices::{IO_REPARSE_TAG_MOUNT_POINT, IO_REPARSE_TAG_SYMLINK}
}
```

But to be honest, I don't know if it's a good practise, I'm failed to find out good practise for it, just think maybe we can reduce `#[cfg(windows)]` usage...

Reply:
Good idea! I’ll try that and your other suggestion, I agree that it would be nice to have fewer cfg lines

Reply:
I think this is unnecessary now that I've moved Windows code into a module, but I'll keep that idea in mind for the future.